NuGet/Xamarin.Microsoft.Band.Native.nuspec

```diff
@@ -41,10 +41,15 @@
         <file src="Component/License.md" target="License.md" />
 		
         <file src="Component/Details.md" target="lib\netcore451\Readme.md" />
+	<file src="NuGet/_._" target="lib\netcore451\_._" />
         <file src="Component/Details.md" target="lib\portable-net45+win+wpa81\Readme.md" />
+	<file src="NuGet/_._" target="lib\portable-net45+win+wpa81\_._" />
         <file src="Component/Details.md" target="lib\portable-win81+wpa81\Readme.md" />
+	<file src="NuGet/_._" target="lib\portable-win81+wpa81\_._" />
         <file src="Component/Details.md" target="lib\uap10.0\Readme.md" />
+    <file src="NuGet/_._" target="lib\uap10.0\_._" />
         <file src="Component/Details.md" target="lib\wpa81\Readme.md" />
+	<file src="NuGet/_._" target="lib\wpa81\_._" />
```

Comment:
Are these really needed? I am using the readme.md for that. Unless the _._ is necessary?